coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -318,57 +275,62 @@ export default {
     cancelNewDirector: function () {
       this.showNewDirectorForm = false
       this.$refs.newDirectorForm.reset()
+      this.$refs.baseAddressNew.$refs.addressForm.reset()
     },
 
-    deleteDirector: function (director, index) {
-      if (this.directors[index] === director) {
-        this.directors.splice(index, 1)
+    deleteDirector: function (director, id) {
+      if (this.directors[id] === director) {
+        this.directors.splice(id, 1)
       } else {
         let found = this.directors.indexOf(director)
         this.directors.splice(found, 1)
-        this.directorsChangeCount--
       }
+      this.activeIndex = null
     },
 
-    validateNewDirectorForm: function (index) {
-      if (this.$refs.newDirectorForm.validate()) {
+    validateNewDirectorForm: function () {
```

Comment:
Does the parent page (AR) need some additional logic to compute if it's validated (similar to AGM Date logic)?